### DISCUSSION TITLE
Adding `stylex.create` inside Storybook render

### DISCUSSION THREAD

**Discussion by tounsoo** [facebook/stylex#514] [Q&A] [answered]
Hello, for better documentation, I would like to add the `stylex.create` inside the render so that consumers can see how it works.

## Sample Story
```tsx
export const OverrideStyle: Story = {
  render: () => {
    const customStyle = stylex.create({
      test: {
        minWidth: "500px",
      },
    });
    
    return <Button style={customStyle.test} >Custom Style</Button>;
  },
};
```

Above returns our friendly error message `stylex.create() is only allowed at the root of a program.`

Is there a way to resolve this error for Storybook example? 

---

## Current status of the Storybook 
I want the `customStyle` piece from above to show in the code section. It currently only works when I have it defined on the root.
<img width="1013" alt="image" src="https://github.com/facebook/stylex/assets/8681348/af1e28cb-c153-4f44-b27f-b1411c7db2a7">

**Comment by nmn** [comment, marked as answer]
This is a constraint of how the compiler works. However, you should be able to write a Babel transform that hoists the `stylex.create` calls out at compile time. This *should* solve your problem.
